src/std_bridge/string.rs

```diff
@@ -79,4 +81,15 @@ impl RustStr {
             len: str.len(),
         }
     }
+
+    pub fn partial_eq(
```

Comment:
Should move this into a `PartialEq` trait implementation.

```rust
impl PartialEq for RustStr {
    // ...
}
```

https://doc.rust-lang.org/std/cmp/trait.PartialEq.html

Reply:
>Should move this into a PartialEq trait implementation. 

Sorry...I know that I should do it, but I want to solve link error. So, I pushed 03245ec39c4cf40f7909704456617f6186f1bc50 to share this problem.